### ISSUE TITLE
GLB does not validate since image contains both a bufferView and URI

### ISSUE THREAD

**Issue by inewland53** [sergkr/gltflib#58]
Currently when exporting a GLB the `gltf.model.images` entries that already have a URI assigned receive a bufferView automatically. This causes issues with the 2.0 spec validation since an Image cannot have both a URI and BufferView at the same time.

I tracked this down in the `gltf.py` script to the `_update_model_after_embedding_resource` method.

It currently only removes the `image.uri` entries for `Base64Resources` but seems like it should also do that for FileResources as well.

Current code:
```python
if image.uri == resource.uri:
    image.bufferView = self._create_embedded_image_buffer_view(offset, bytelen)
    # If the resource is a Base64 resource, then delete the image URI so we're not representing the
    # data twice, and preserve the MIME type.
    if isinstance(resource, Base64Resource):
        image.uri = None
        image.mimeType = resource.mime_type
```

A quick test adding `image.uri = None` fixed my situation:
```python
if image.uri == resource.uri:
    image.bufferView = self._create_embedded_image_buffer_view(offset, bytelen)
    # If the resource is a Base64 resource, then delete the image URI so we're not representing the
    # data twice, and preserve the MIME type.
    if isinstance(resource, Base64Resource):
        image.uri = None
        image.mimeType = resource.mime_type
    elif isinstance(resource, FileResource):
        image.uri = None
```

I apologize I have limited knowledge of the GLTF/GLB spec. Does this fix make sense or did I do something wrong when setting up my GLTF object?

**Comment by Vochsel** [comment]
Jumping on to also say I had this issue, and this fix solved my issue. Thanks!

Any word on getting this released?

**Comment by sergkr** [comment]
Sorry about my hiatus - this has now been fixed and released in v1.0.6.
